src/dynamics/rigid_body/mod.rs

```diff
@@ -431,13 +431,19 @@ pub(crate) struct PreSolveAngularVelocity(pub Vector);
 /// }
 /// ```
 #[derive(
-    Component, Reflect, Debug, Clone, Copy, PartialEq, PartialOrd, Default, Deref, DerefMut, From,
+    Component, Reflect, Debug, Clone, Copy, PartialEq, PartialOrd, Deref, DerefMut, From,
 )]
 #[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
 #[cfg_attr(feature = "serialize", reflect(Serialize, Deserialize))]
 #[reflect(Debug, Component, Default, PartialEq)]
 pub struct GravityScale(pub Scalar);
 
+impl Default for GravityScale{
+    fn default() -> Self {
+        Self(1.0f32)
```

Comment:
`1.0f32` will break `f64` since `GravityScale` currently stores a `Scalar`, which is `f32`/`f64` depending on the feature flag.

```suggestion
        Self(1.0)
```